lib/trello/json_utils.rb

```diff
@@ -47,8 +48,13 @@ def from_json(json)
         end
       end
 
-      def parse_json(string, encoding = 'UTF-8')
-        JSON.parse(string.force_encoding(encoding))
+      def parse_json(data, encoding = 'UTF-8')
+        case data
+        when Faraday::Response
+          JSON.parse(data.body.force_encoding(encoding))
+        else
+          JSON.parse(data.force_encoding(encoding))
+        end
```

Comment:
I realize this might not be the most ideal place to parse out a Faraday response body, but it's the place that had the least impact on the rest of the code. I'm open to moving this if we'd rather it not be here.